### ISSUE TITLE
question about accuracy

### ISSUE THREAD

**Issue by pengzhiliang** [facebookresearch/classifier-balancing#11]
Thanks for the great work!

I have a question. Is the accuracy reported in your paper on the validation set or the test set? If it is on the validation set, but I use the resnet50 pretrainmodel of the first stage you provided on github, and the second stage using crt and lws can reach 48.0%, which is slightly higher than the results in the paper. But if it is on the test set, the results of using crt and lws in the second stage are 46.3% and 46.6%, respectively, which is little far from the results reported in the paper. I am very confused about this question.

**Comment by pengzhiliang** [comment]
The dataset is ImageNet-LT

**Comment by pengzhiliang** [comment]
OK, I know it. According to the second stage weights([Renext50 crt](https://dl.fbaipublicfiles.com/classifier-balancing/ImageNet_LT/clslearn/resnext50_crt_uni2bal.pth)) you provided on github, I tested its accuracy on the test set, and the output is as follows:
```bash
Phase: test
100%|████████████████████████████████████████████████████████████████████████| 98/98 [01:36<00:00,  1.12it/s]


 Phase: test 

 Evaluation_accuracy_micro_top1: 0.481 
 Averaged F-measure: 0.467 
 Many_shot_accuracy_top1: 0.602 Median_shot_accuracy_top1: 0.445 Low_shot_accuracy_top1: 0.266 

60.2     44.5    26.6    48.1
```
which is very different from Table 7 in the paper

61.8 | 46.2 | 27.4 | 49.6
-- | -- | -- | --

Therefore, the accuracy in the paper in on validation set
